crates/solvers/openapi.yml

```diff
@@ -704,3 +742,27 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Interaction"
+        score:
+          description: |
+            Information required by the driver to score the solution. 
+            
+            Can either be a raw score or a revert probability.
```

Comment:
I think we are expecting the success probability here (which is also more in line with your example of `0.9`).